.github/workflows/ci.yml

```diff
@@ -25,3 +25,9 @@ jobs:
         run: sbt scripted
       - name: Test generate documentation
         run: sbt doc
+  ci:
```

Comment:
You can designate `ci` as the required job for a PR to pass

Reply:
It is not clear to me what is the difference with the `build` job, could you please elaborate?

Reply:
You have now multiple jobs for CI (one for 8, one for 11, one for 17). You don't want to be mark each one of them as _required_ (and changing that every time you change GitHub Actions settings).

It's better to have just _one_ job (`ci` in this case) that aggregates/depends on all of them. Mark only that one as _required_.

Reply:
OK I see, thank you for the explanation!